system/Database/SQLSRV/Forge.php

```diff
@@ -276,51 +293,66 @@ protected function _processColumn(array $field): string
      *
      * @param string $table Table name
      */
```

Comment:
Added second parameter $asQuery. This tells the function to return a stand alone query rather than the partial sql used when creating a table. Also returns array instead of string.